ckanext/ontario_theme/templates/internal/group/read_base.html

```diff
@@ -1,5 +1,12 @@
 {% ckan_extends %}
 
+{% block meta_description -%}
+  {% if h.get_translated(c.group_dict,"description") %}
+    <meta name="description"
+          content="{{ h.markdown_extract(h.get_translated(c.group_dict,'description'), 165) }}">
```

Comment:
Should 165 be hard-coded here, or stored as a variable somewhere?

Reply:
Google truncates the tags themselves at around 155-160 characters. 165 is just so our description tags aren't super long. Technically we don't need the hard coded. Should we just remove them?

Reply:
markdown_extract has a default truncation of 190: https://docs.ckan.org/en/2.9/theming/template-helper-functions.html#ckan.lib.helpers.markdown_extract
so we remove `extract_length=165` entirely